src/main/java/macrobase/ingest/SQLLoader.java

```diff
@@ -88,6 +93,7 @@ public RowSet getRows(String baseQuery,
                           List<RowSetResource.RowSetRequest.RowRequestPair> preds,
                           int limit,
                           int offset) throws SQLException {
+        // TODO do we need to handle time column here?
```

Comment:
Do we need to order by the time column here as well? Also, the approach in this file seems rather hacky in general - I'm currently assuming `baseQuery` doesn't already contain the `ORDER BY` etc etc. But I guess it's good enough for our purposes for now.